tests/test_zeus_client.py

```diff
@@ -68,23 +81,22 @@ def test_validate_dates(self):
     @mock.patch('zeus.interfaces.rest.requests')
     def test_post_empty_log(self, mock_requests):
         logs = []
+        url = '{}/logs/{}/ZeusTest'.format(FAKE_SERVER, FAKE_TOKEN)
```

Comment:
if user doesn't set http or https, requests library will fail anyway...

i;m only saying this because i had bug in the past that were caused by the client adding an extra `/` at the end of the url:

`http://localhost:8000`  works
`http://localhost:8000/` doesn't work

The error message that arises from that is non obvious about the forward slash and will take time to track when it happens, i'm very wary of handling urls or paths manually because of that.